public/intro/2_Lecture_2.md

```diff
@@ -36,6 +41,7 @@ As a rule of thumb for precedence, binary arithmetic operators (e.g. +) bind mor
 | 15         | \|\|                     | Left-to-right |
 | 16         | =                        | Right-to-left |
 
+
```

Comment:
Why are you adding this line here ? I don't think that it's necessary

Reply:
44?